pipelines/build/common/openjdk_build_pipeline.groovy

```diff
@@ -395,36 +395,28 @@ class Build {
     */
     def sign(VersionInfo versionInfo) {
         // Sign and archive jobs if needed
-        // TODO: This version info check needs to be updated when the notarization fix gets applied to other versions.
         if (
-            buildConfig.TARGET_OS == "windows" ||
-        (buildConfig.TARGET_OS == "mac" && versionInfo.major == 8 && buildConfig.VARIANT != "openj9") || (buildConfig.TARGET_OS == "mac" && versionInfo.major == 13)
+            buildConfig.TARGET_OS == "windows" || (buildConfig.TARGET_OS == "mac" && versionInfo.major == 8)
```

Comment:
logic is changed from above - is that really correct (excluding the removal of openj9)?

Reply:
yes, we now want to use the previous signing method for jdk8 (hotspot and J9)